docs/eloquent-models.txt

```diff
@@ -6,517 +6,23 @@ Eloquent Models
 
 .. facet::
    :name: genre
-   :values: tutorial
+   :values: reference
 
 .. meta::
-   :keywords: php framework, odm, code example
+   :keywords: php framework, odm
 
-This package includes a MongoDB enabled Eloquent class that you can use to
-define models for corresponding collections.
+Eloquent models are part of the Laravel Eloquent object-relational
+mapping (ORM) framework that enable you to work with a database by using
+model classes. The {+odm-short+} extends this framework to use similar
+syntax to work with MongoDB as a database.
 
-Extending the base model
-~~~~~~~~~~~~~~~~~~~~~~~~
+This section contains guidance on how to use Eloquent models in
+{+odm-short+} to work with MongoDB in the following ways:
 
-To get started, create a new model class in your ``app\Models\`` directory.
+- :ref:`laravel-schema-builder` shows how to manage indexes on your MongoDB
+  collections
```

Comment:
Question: Should we mention that this page talks about migrations too?

```suggestion
- :ref:`laravel-schema-builder` shows how to perform migrations and manage indexes on your MongoDB
  collections
```

Reply:
I'll update this to "manage indexes on your collections by using migrations" since the content doesn't focus on migrations, but rather using them as a means to perform the index management (technically, you could use a different mechanism to manage the indexes).